modules/ossm-cr-pilot.adoc

```diff
@@ -5,7 +5,7 @@
 [id="ossm-cr-pilot_{context}"]
 = Istio Pilot example
 
-Here is an example that illustrates the Istio Pilot parameters for the {ProductName} custom resource and a description of the available parameters with appropriate values.
+Here is an example that illustrates the Istio Pilot parameters for the `ServiceMeshControlPlane` and a description of the available parameters with appropriate values.
 
 [source,yaml]
 ----
```

Comment:
Indentation in the `autoscaleEnabled` and `traceSampling` lines is off.

Reply:
@luksa How so? They're both indented at the same level, which is identical to the example on the Maistra site.

Reply:
GitHub link to the file from here isn't working for some reason, but the preview looks right to me.  
pilot:
    resources:
      requests:
        cpu: 100m
     autoscaleEnabled: false
     traceSampling: 100.0

Reply:
Actually, I think I see it. 

```
pilot:
   resources:
     requests:
       cpu: 100m
    autoscaleEnabled: false
    traceSampling: 100.0
```
Pilot to resources is two spaces. Resources to requests is two spaces. Pilot to autoscaleEnabled and traceSampling looks like it's three spaces.

Reply:
Fixed.